### DISCUSSION TITLE
Wrong button output in most games

### DISCUSSION THREAD

**Discussion by InvisibleRasta** [nefarius/DsHidMini#309] [Support]
Hello, I am on a fresh install of windows 11. First time using dshidmini.
The problem I am experiencing is that the buttons do not respond correctly in games. For example in the game Wreckfest X button is LB and many of the other buttons do not respond.
On my previous windows 11 install I was using ScpToolkit and everything was working as expected.
I did follow the documentation to install. And being a new install I do not have any other drivers installed. The controller is original.
![ss](https://github.com/nefarius/DsHidMini/assets/30435428/4c18be4c-1e47-46e1-879e-fda373b41c9e)

**Comment by nefarius** [comment]
> I did follow the documentation to install.

Then please simply read the rest of the documentation too, these exact issues are all explained in great detail there.

**Reply by InvisibleRasta** [reply]
I am not sure on what part of the documentation you are talking about. I went trought the documentation looking for an answer before coming here. I have seen a bug report suggesting to chane the device mode to SDF. I tried that alredy and no buttons respond at all.

**Reply by nefarius** [reply]
[This is a prime example](https://docs.nefarius.at/projects/DsHidMini/HID-Device-Modes-Explained/), **you** need to configure your setup properly, this is not a bug. [Also the FAQ](https://docs.nefarius.at/projects/DsHidMini/FAQ/).

**Reply by InvisibleRasta** [reply]
I am not really sure what hid mode i need to use after reading the guide. I know i have a original PS3 dualshock controller and that with scptoolkit it worked out of the box without any fiddling.
I got it to work with DS4Windows but it seems it will work only when the software is running

**Reply by nefarius** [reply]
> I got it to work with DS4Windows but it seems it will work only when the software is running

Well, yes, that is working as intended.

**Reply by InvisibleRasta** [reply]
it has to keep running in the taskbar?

**Reply by nefarius** [reply]
Yes?

**Reply by InvisibleRasta** [reply]
oh this is so different than scptoolkit. Never had to have it running to use it. I guess scptoolkit worked in the background

**Reply by nefarius** [reply]
ScpToolkit constantly ran multiple processes in the background, so does this solution, if you want functionality, programs have to run. Computing basics.

**Reply by InvisibleRasta** [reply]
can i have this program in the background? manually opening it every time i need it is kind of a pain

**Reply by nefarius** [reply]
Ofc. you can but this is outside of the scope of my craft, use a search engine of your liking to figure it out, good luck.

**Reply by InvisibleRasta** [reply]
thank you for your kindness.
